### PR TITLE
rename doc and function, add new interface of like

### PR DESCRIPTION
**Pull request by VictorZhou123**
1. rename some function
2. annotation function that shall be remaked
3. remake wukong doc
4. add like serise interface
    1. add picture into my like from generated
    2. add picture into my like from public picture
    3. list picture of my like
    4. cancel like picture

### CODE REVIEW

File: infrastructure/mongodb/wukong_picture.go [hunk 1]
```diff
@@ -73,7 +114,7 @@ func (col wukongPicture) newDoc(user string) error {
 
 	doc := bson.M{
 		fieldOwner:   user,
-		fieldItems:   bson.A{},
+		fieldLikes:   bson.A{},
```
Comment (zengchen1024) [inline infrastructure/mongodb/wukong_picture.go:117]: missing fieldPublics?

File: infrastructure/mongodb/wukong_picture.go [hunk 2]
```diff
@@ -128,12 +174,15 @@ func (col wukongPicture) Delete(user string, pid string) error {
...
+func (col wukongPicture) getByUserName(user, pid, field string) (
+	do repositories.WuKongPictureDO,
+	err error,
+) {
 	var v []dWuKongPicture
 
 	f := func(ctx context.Context) error {
 		return cli.getArrayElem(
-			ctx, col.collectionName, fieldItems,
+			ctx, col.collectionName, fieldLikes,
```
Comment (zengchen1024) [inline infrastructure/mongodb/wukong_picture.go:185]: should it be field instead of fieldLike?

**Review by zengchen1024 — CHANGES_REQUESTED** [review]

File: infrastructure/mongodb/wukong_picture.go (hunk 1 above)
Comment (VictorZhou123) [reply infrastructure/mongodb/wukong_picture.go:117]: fixed @zengchen1024

File: infrastructure/mongodb/wukong_picture.go (hunk 2 above)
Comment (VictorZhou123) [reply infrastructure/mongodb/wukong_picture.go]: fixed

File: infrastructure/mongodb/doc.go [hunk 3]
```diff
@@ -491,35 +493,32 @@ type luojiaItem struct {
...
-	Style string `bson:"style"           json:"style"`
-}
-
 type dWuKongPicture struct {
 	Owner   string `bson:"owner"         json:"owner"`
 	Version int    `bson:"version"       json:"-"`
 
-	Items []pictureItem `bson:"items"    json:"-"`
+	Likes   []pictureItem `bson:"likes"    json:"-"` // like picture
+	Publics []pictureItem `bson:"publics" json:"-"`  // public picture
```
Comment (zengchen1024) [inline infrastructure/mongodb/doc.go:510]: please Align codes

File: infrastructure/mongodb/doc.go [hunk 4]
```diff
@@ -491,35 +493,32 @@ type luojiaItem struct {
...
-	Desc      string `bson:"desc"        json:"desc"`
-	Style     string `bson:"style"       json:"style"`
-	OBSPath   string `bson:"obspath"     json:"obspath"`
-	CreatedAt string `bson:"created_at"  json:"created_at"`
+	Id        string   `bson:"id"          json:"id"`
+	Desc      string   `bson:"desc"        json:"desc"`
+	Style     string   `bson:"style"       json:"style"`
+	OBSPath   string   `bson:"obspath"     json:"obspath"`
+	Diggs     []string `bson:"diggs" json:"diggs"`
+	DiggCount int      `bson:"digg_count" json:"digg_count"`
```
Comment (zengchen1024) [inline infrastructure/mongodb/doc.go:519]: ditto

**Review by zengchen1024 — CHANGES_REQUESTED** [review]

File: infrastructure/mongodb/doc.go (hunk 3 above)
Comment (VictorZhou123) [reply infrastructure/mongodb/doc.go]: fixed @zengchen1024

File: infrastructure/mongodb/doc.go (hunk 4 above)
Comment (VictorZhou123) [reply infrastructure/mongodb/doc.go]: fixed

**Pull request merged**